packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -6,40 +6,54 @@ import {
     ERC721BurnableUpgradeable
 } from "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
 import { AttestationStation } from "./AttestationStation.sol";
+import { OptimistAllowlist } from "./OptimistAllowlist.sol";
 import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 
 /**
  * @author Optimism Collective
  * @author Gitcoin
- * @title Optimist
+ * @title  Optimist
  * @notice A Soul Bound Token for real humans only(tm).
  */
 contract Optimist is ERC721BurnableUpgradeable, Semver {
+    /**
+     * @notice Attestation key used by the attestor to attest the baseURI.
+     */
+    bytes32 public constant BASE_URI_ATTESTATION_KEY = bytes32("optimist.base-uri");
+
+    /**
+     * @notice Attestor who attests to baseURI.
+     */
+    address public immutable BASE_URI_ATTESTOR;
```

Comment:
Unrelated to pr: In general I think using the attestation station for the base uri is unnecessary complexity we should remove

Reply:
Yeah, I didn't have context on this decision. Was this in order to showcase a use for the AttestationStation?

Reply:
Yea I think we were just attempting to use attestations anywhere it could be used even if it didn't make sense

Reply:
Pretty sure @smartcontracts lightly recommended removing it too but we didn't just because it would require some offchain code in @gateway/backend to change too